analyzer/codechecker_analyzer/analysis_manager.py

```diff
@@ -783,6 +785,7 @@ def signal_handler(signum, frame):
                          analyzer_config_map.get(build_action.analyzer_type),
                          output_path,
                          skip_handlers,
+                         rs_handler,
```

Comment:
Yes, I considered similar solutions. For example AnalyzerContext is a singleton in CodeChecker which is available everywhere in the analyzer without wiring it through such monster functions with >10 parameters. The simplification of these functions is among the plans.
I also considered handling "skipfile" and "review status" config by the same class, because the "ignore" option of the review status config file has a similar "skip" effect.
But in the end I chose using the same pattern what skip handler uses. I can tell you later why I chose this instead of the other options.